pylanetary/navigation/core.py

```diff
@@ -686,11 +690,7 @@ def xy_shift_model(self, dx, dy):
             [pixels] shift in y
         '''
         shape = self.data.shape
-        xcen, ycen = int(shape[0]/2), int(shape[1]/2) #pixels at center of planet
-        yy = np.arange(shape[0]) - xcen - dy
-        xx = np.arange(shape[1]) - ycen - dx
-        x,y = np.meshgrid(xx,yy) 
-        self.lat_g, self.lat_c, self.lon_w = lat_lon(x,y,self.ob_lon,self.ob_lat,self.pixscale_km,self.np_ang,self.req,self.rpol)
+        self.lat_g, self.lat_c, self.lon_w, self.image_grid_km_x, self.image_grid_km_y = lat_lon(shape,self.pixscale_km,self.ob_lon,self.ob_lat,self.np_ang,self.req,self.rpol,dx=dx,dy=dy)
```

Comment:
I've seen in our linter called ruff that these are now preferred to be stacked vertically when there are long inputs. It's a style thing, just a heads up. 

It would be:
lat_lon(shape, 
             self.pixescale_km, 
             self.ob_lon........

Reply:
I hit the whole thing with autopep8, should have fixed this and other instances